specification/compute/resource-manager/Microsoft.Compute/stable/2019-07-01/gallery.json

```diff
@@ -101,6 +101,55 @@
           }
         }
       },
+      "patch": {
+        "tags": [
+          "Galleries"
+        ],
+        "operationId": "Galleries_Update",
+        "description": "Update a Shared Image Gallery.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "galleryName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Shared Image Gallery. The allowed characters are alphabets and numbers with dots and periods allowed in the middle. The maximum length is 80 characters."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "gallery",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/GalleryUpdate"
+            },
+            "description": "Parameters supplied to the update Shared Image Gallery operation."
+          }
+        ],
+        "responses": {
+          "200": {
```

Comment:
For long running operation, why the response code is 200?

Reply:
This was a decision made by all the compute RPs. In the past 202 would have been returned but for a year now all the CompuuteRPs return 200 and the user polls via location/azyre-async polling.